README.md

```diff
@@ -466,6 +466,35 @@ To build an Amazon-like autocomplete menu, suggesting popular queries and for th
 }
 ```
 
+##### User-generated data: protecting against XSS
+
+Malicious users may attempt to engineer XSS attacks by storing HTML/JS in their data. It is important that user-generated data be properly escaped before using it in an autocomplete.js template.
+
+The `enableXSSProtection` option can be set to `true` to provide an `escapeHTML` function as third argument to the suggestion template function. It can be used in the following way:
+
+```js
+$('#states').autocomplete({}, [
+  {
+    displayKey: 'name',
+    source: $.fn.autocomplete.sources.hits(index, {
+      hitsPerPage: 5,
+      enableXSSProtection: true,
+      highlightPreTag: "<strong style=\"red\">",
```

Comment:
style=red will not trigger anything valid. I would just put "<strong></strong>", should be enough for understanding.